my_enumerable.rb 

```diff
@@ -0,0 +1,35 @@
+module MyEnumerable
+  def list
+    @list ||= []
+  end
+
+  def all?
+    return unless block_given?
```

Comment:
- [ ] `my_enumerable.rb` has a trailing white space and therefore generates an error. Kindly fix it.